### PR TITLE
Fix small grammar issues

### PR DESCRIPTION
**Pull request by vosmith**
This make a few corrections to the README, mostly focused on verb tense and articles.

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -413,7 +413,7 @@ $ docker run -p 8080:8080 -it process-quarkus-example
 # On another shell do a simple post request 
 curl -d '{"approver" : "john", "order" : {"orderNumber" : "12345", "shipped" : false}}' -H "Content-Type: application/json" -X POST http://localhost:8080/orders
 
-#repare the container logs the following message:
+# the container logs the following message:
```
Comment (spolti) [inline README.md]:
```suggestion
# notice the container logs the following message:
```

File: README.md [hunk 2]
```diff
@@ -434,7 +434,7 @@ $ docker run -p 8080:8080 -it process-quarkus-example-non-uberjar
 # On another shell do a simple post request 
 $ curl -d '{"approver" : "john", "order" : {"orderNumber" : "12345", "shipped" : false}}' -H "Content-Type: application/json" -X POST http://localhost:8080/orders
 
-#repare the container logs the following message:
+# the container logs the following message:
```
Comment (spolti) [inline README.md]: same here

File: README.md [hunk 3]
```diff
@@ -466,7 +466,7 @@ $ docker run -it -p 8080:8080 spring-binary-example
 # on another terminal, interact with the kogito service
 $ curl -d '{"approver" : "john", "order" : {"orderNumber" : "12345", "shipped" : false}}' -H "Content-Type: application/json" -X POST http://localhost:8080/orders
 
-#repare the container logs the following message:
+# the container logs the following message:
```
Comment (spolti) [inline README.md]: and here.

File: README.md [hunk 4]
```diff
@@ -512,15 +512,15 @@ $ docker run -it -p 8080:8080 binary-test-example
...
```
Comment (spolti) [inline README.md]: and here

**Review by spolti — CHANGES_REQUESTED** [review]
Hi, many thanks for your contribution, just a few things that can be improved,
thanks!

**Comment by vosmith** [comment]
@spolti, thanks for the review!  All comments should be addressed now.

**Review by spolti — APPROVED** [review]

**Comment by spolti** [comment]
Thanks @vosmith, we really appreciate it.

**Pull request merged**